ecosystem/typescript/sdk/src/plugins/aptos_token.ts

```diff
@@ -484,7 +501,50 @@ export class AptosToken {
       [HexString.ensure(token).hex(), HexString.ensure(recipient).hex()],
     );
     const bcsTxn = AptosClient.generateBCSTransaction(owner, rawTxn);
-    const pendingTransaction = await this.provider.aptosClient.submitSignedBCSTransaction(bcsTxn);
+    const pendingTransaction = await this.provider.submitSignedBCSTransaction(bcsTxn);
     return pendingTransaction.hash;
   }
+
+  /**
+   * Transfer a token. This function supports transfer non-fungible token and fungible token.
+   *
+   * To set the token type, set isFungibleToken param to true or false.
+   * If isFungibleToken param is not set, the function would query Indexer
+   * for the token data and check whether it is a non-fungible or a fungible token.
+   *
+   * @param data NonFungibleTokenParameters | FungibleTokenParameters type
+   * @param isFungibleToken (optional) The token type, non-fungible or fungible token.
+   * @returns The hash of the transaction submitted to the API
+   */
+  async transfer(
+    data: NonFungibleTokenParameters | FungibleTokenParameters,
+    isFungibleToken?: boolean | null,
+  ): Promise<string> {
```

Comment:
Does this work for token v1?

Reply:
no, `aptos_token` is for token v2. `token_client` is for token v1.

Reply:
I think this is a good callout for token v1 confusion, but I think we should be moving towards the ideal experience for token V2